example/package.json

```diff
@@ -56,6 +56,7 @@
     "@cumulus/files-to-granules": "1.21.0",
     "@cumulus/hello-world": "1.21.0",
     "@cumulus/integration-tests": "1.21.0",
+    "@cumulus/message": "1.21.0-alpha1",
```

Comment:
Quick question while I'm thinking about it - did we settle on doing alphas instead of just current minor for new packages?

Reply:
No, I'm just doing alphas until the code is approved in case there are bugs in the code, then I'll publish as 1.21.0 and update the PR

Reply:
👍